tests/emsi/parsers/test_skill_parsers.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for the `taxonomy-connector` emsi data parsers.
+"""
+import ddt
+
+from taxonomy.emsi.parsers.skill_parsers import SkillDataParser
+from test_utils.testcase import TaxonomyTestCase
+
+
+@ddt.ddt
+class TestSkillDataParser(TaxonomyTestCase):
+    """
+    Validate that JWT token are fetched and cached appropriately.
```

Comment:
nit: this docstring doesn't seem relevant to the testcase class

Reply:
Copy-Paste issue :)